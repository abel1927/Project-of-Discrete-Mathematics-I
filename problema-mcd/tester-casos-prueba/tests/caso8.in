50
9311702399 9311702400
9311702399 9311702400
9311702399 9311702400
9311702399 9311702400
9311702399 9311702400
9311702399 9311702400
9311702399 9311702400
9311702399 9311702400
9311702399 9311702400
9311702399 9311702400
9311702399 9311702400
9311702399 9311702400
9311702399 9311702400
9311702399 9311702400
9311702399 9311702400
9311702399 9311702400
9311702399 9311702400
9311702399 9311702400
9311702399 9311702400
9311702399 9311702400
9311702399 9311702400
9311702399 9311702400
9311702399 9311702400
9311702399 9311702400
9311702399 9311702400
9311702399 9311702400
9311702399 9311702400
9311702399 9311702400
9311702399 9311702400
9311702399 9311702400
9311702399 9311702400
9311702399 9311702400
9311702399 9311702400
9311702399 9311702400
9311702399 9311702400
9311702399 9311702400
9311702399 9311702400
9311702399 9311702400
9311702399 9311702400
9311702399 9311702400
9311702399 9311702400
9311702399 9311702400
9311702399 9311702400
9311702399 9311702400
9311702399 9311702400
9311702399 9311702400
9311702399 9311702400
9311702399 9311702400
9311702399 9311702400
9311702399 9311702400
