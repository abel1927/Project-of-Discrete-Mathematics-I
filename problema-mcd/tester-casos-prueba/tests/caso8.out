1592524800
1592524800
1592524800
1592524800
1592524800
1592524800
1592524800
1592524800
1592524800
1592524800
1592524800
1592524800
1592524800
1592524800
1592524800
1592524800
1592524800
1592524800
1592524800
1592524800
1592524800
1592524800
1592524800
1592524800
1592524800
1592524800
1592524800
1592524800
1592524800
1592524800
1592524800
1592524800
1592524800
1592524800
1592524800
1592524800
1592524800
1592524800
1592524800
1592524800
1592524800
1592524800
1592524800
1592524800
1592524800
1592524800
1592524800
1592524800
1592524800
1592524800
